src/app/compiler.js

```diff
@@ -207,9 +212,11 @@ function Compiler (editor, handleGithubCall) {
   function gatherImports (files, importHints, cb) {
     importHints = importHints || []
     if (!compilerAcceptsMultipleFiles) {
-      cb(files[editor.getCacheFile()])
+      cb(null, files[editor.getCacheFile()])
       return
     }
+    // FIXME: This will only match imports if the file begins with one.
```

Comment:
Note that this is only relevant for compiler versions that do not report missing inputs.